src/components/common/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import styles from './DatePicker.module.css';
+import CalendarIcon from '#/icons/icon-calendar.svg';
+
+interface Props {
+  value: string;
+  onClick?: () => void;
+}
+
+const CustomInput = ({ value, onClick }: Props) => (
+  <label className={styles.container}>
+    <button className={styles.input} onClick={onClick}>
+      {value === '' ? <span>{value || 'YY/MM/DD'}</span> : <>{value || 'YY/MM/DD'}</>}
+    </button>
```

Comment:
```suggestion
    <button className={styles.input} onClick={onClick}>
 {value ? <>{value}</> : <span>{'YY/MM/DD'}</span>}    </button>
```
value의 존재 여부를 앞의 조건문에서 확인한 뒤에 또한 '||' 연산자로 확인할 필요가 없습니다. 또한, 없을 때를 삼항 연산자의 조건문으로 제시하는 것보다는 있을 때를 조건문으로 제시하는 것이 직관적이라고 생각합니다. 즉, '값이 있으면'의 수행할 표현식을 : 앞에 표시하고, '값이 없으면' 수행할 표현식을 뒤에 표시하는 것이 더 직관적으로 이해하기 쉽다는 것입니다